### PR TITLE
Feature/fix settings form regression

### LINKED ISSUE #1474: Restructure settings module
The settings module code does not follow the structure of other project modules. Re-structure the Settings module to follow the [project structure conventions](https://github.com/apinf/api-umbrella-dashboard/blob/develop/CONTRIBUTING.md#file-structure).

# original issue
The Settings form does not work in `develop`. This may be due to switching the collection to using `export` / `import` syntax. Find the underlying issue and fix it.

### PR DESCRIPTION
**Pull request by brylie**
Closes #1474 

# Changes
- Import Settings collection in settings file(s)
- Create template helper for settingsCollection, used by AutoForm
- Cleanup AutoForm hooks
- Remove unneeded Settings schema fields
- Automatic linting
- Flash error reason in AutoForm hook

### CODE REVIEW

**Comment by brylie** [comment]
@marla-singer this PR should fix the Settings form. Would you please review?

**Comment by marla-singer** [comment]
Button "Save" is become disable on error, if you input incorrect data, and doesn't become undisable.
joxi_screenshot_1472716883705

**Comment by marla-singer** [comment]
Browser console has next output on main page:
```
Exception in template helper: ReferenceError: Settings is not defined
    at Object.contactDetailsValid
```
Server console has this output:

```
Exception while invoking method 'isInitialSetupComplete' ReferenceError: Settings is not defined
at [object Object].isInitialSetupComplete (setup/server/methods.js:6:22)

```
I think it happens because of import of Settings missing in these files

File: settings/client/autoform.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+AutoForm.hooks({
+  settingsForm: {
+    onSuccess () {
+      FlashMessages.sendSuccess('Settings saved.');
```
Comment (marla-singer) [inline settings/client/autoform.js]: Maybe message text should be added in i18n token?

Comment (brylie) [inline settings/client/autoform.js]: Jep. Thanks.

**Comment by brylie** [comment]
@marla-singer I have fixed the console errors and merged you SDK schema changes. Please review and merge if there are no further issues.

**Comment by marla-singer** [comment]
@brylie I add import of Settings in file documentation/client/view/documentation.js and SDK works good. Have I do push this remark or you do a change in your code?

**Comment by marla-singer** [comment]
@brylie Unresolve problem with disable button 'Save'

**Step to reproduce:**

1. Go to Settings page
2. Click checkbox 'Enabled' of SDK Code Generator
3. Leave empty input 'Host'
4. Click button 'Save'

Button is become disabled and saves this position, and client console has output:

```
Uncaught RangeError: Maximum call stack size exceeded collection.js:535 
```

**Comment by brylie** [comment]
I opened an [upstream support request](https://github.com/aldeed/meteor-autoform/issues/1514). I will try other AutoForms in our project, to see if they throw errors when validating.

**Comment by brylie** [comment]
Hm. We can close this branch for now. I will create a task to refactor the Settings module to follow the project module structure.

**Pull request closed without merging**